src/FormattedText.tsx

```diff
@@ -1,12 +1,36 @@
-import * as Marked from 'marked';
+import * as MarkdownIt from 'markdown-it';
 import * as React from 'react';
-import * as He from 'he';
+
+export interface MarkdownItOptions{
```

Comment:
No need to define this type. It's already in @types/markdown-it: https://www.npmjs.com/package/@types/markdown-it